<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="cerrarAplicativo">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>690</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> Cerrar aplicativo</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Iconos/close.png</normaloff>Iconos/close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="text">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>581</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CORRELACIONES SPEARMANR Y PEARSON</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>541</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>1. Seleccione el archivo y programa a trabajar</string>
    </property>
    <widget class="QPushButton" name="cargarArchivo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>   Seleccionar archivo ...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Iconos/folder.png</normaloff>Iconos/folder.png</iconset>
     </property>
    </widget>
    <widget class="QTextEdit" name="textArchivo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Archivo seleccionado:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>451</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Programa a trabajar</string>
     </property>
     <widget class="QCheckBox" name="ckboxOneSample">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>One sample</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Segundo Ejemplo</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="CkboxCorrelaciones">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Correlación</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>541</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>3. Realizar el análisis</string>
    </property>
    <widget class="QTextEdit" name="resultadoCorrelacion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>501</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Seleccione la cantidad de registros a trabajar</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="SbCantidaRegis">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>3</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>541</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>2. Seleccione las columnas</string>
    </property>
    <widget class="QPushButton" name="Adicionar">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Adicionar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Iconos/add.png</normaloff>Iconos/add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QComboBox" name="ListadoCampos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Listado de campos seleccionados:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textColumnas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>341</width>
       <height>101</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="borrarColumnas">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Borrar Columnas</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Iconos/broom.png</normaloff>Iconos/broom.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="mensajeErrorColumnas">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>No puede agregar mas campos</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="ImagenColores">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>380</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="procesarArchivo">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>350</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Procesar archivo ...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Iconos/work-process.png</normaloff>Iconos/work-process.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1157</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
